packages/open-next/src/adapters/cache.ts

```diff
@@ -517,11 +517,17 @@ export default class S3Cache {
 
   private async deleteS3Objects(key: string) {
     try {
-      const regex = new RegExp(`\.(json|rsc|html|body|meta|fetch|redirect)$`);
+      const regexString = `\.(json|rsc|html|body|meta|fetch|redirect)$`;
```

Comment:
The `\.` needs to be escaped: `\\.`

Reply:
I was actually thinking this...but I figured this wouldve come up before now...